src/pages/settings/NewPasswordForm.js

```diff
@@ -71,8 +72,8 @@ class NewPasswordForm extends React.Component {
                 <TextInput
                     label={`${this.props.translate('setPasswordPage.enterPassword')}`}
                     secureTextEntry
-                    autoCompleteType="password"
-                    textContentType="password"
+                    autoComplete={ComponentUtils.NEW_PASSWORD_AUTOCOMPLETE_TYPE}
+                    textContentType="newPassword"
```

Comment:
So we don't need a native const here because it's the same for all platforms?

Reply:
It's an iOS only prop https://reactnative.dev/docs/textinput#textcontenttype-ios

Reply:
And autoComplete is for web or android.

Hmm, i guess we could rename `.native.js` to `.android.js`

Reply:
Thanks